internal/events/dx_callbacks.go

```diff
@@ -224,6 +211,7 @@ func (em *eventManager) privateBlobReceived(dx dataexchange.Plugin, event dataex
 	}
 	if br.Namespace != em.namespace {
 		log.L(em.ctx).Debugf("Ignoring blob from different namespace '%s'", br.Namespace)
+		event.Ack() // Still confirm the event
```

Comment:
Are we guaranteed to only pass the event to a single DX Callback handler? ... if not, what is the implication that `event.Ack()` could be called multiple times?

Reply:
... I remember we made changes to the event ack interface to FFDX, but I can't remember if that included an ID in the ack such that a duplicate was ignored. Feel like maybe I should check that

Reply:
Yep - all good, the FFDX API was upgraded to ack-by-id:
https://github.com/hyperledger/firefly-dataexchange-https/blob/10b23db379753b83c40da210fb86ceea4226cb42/src/handlers/events.ts#L100-L120